src/components/Dashboard/StaffNotation.vue

```diff
@@ -165,7 +165,29 @@
             </v-list-item>
 
             <v-list-item
-              v-if="!isHistorical && (isBenBcCccUlc || isCoop)"
```

Comment:
I left `isHistorical` here in case of future template refactor. The other logic is in a computed (getter) method below.

Ditto for the other 3 actions here.